nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-api/src/main/java/org/nuxeo/ecm/directory/providers/DirectoryEntryPageProvider.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and others.
```

Comment:
I think the right copyright is just `Nuxeo` and not `Nuxeo SA`

Reply:
I copied it from `BatchHandler`

Reply:
Indeed, replacement wasn't done for all files, we did it in the same time than cleanup and format step.
As we can see on [doc](https://doc.nuxeo.com/corg/copyright-and-license-headers/), desired copyright is indeed `Nuxeo`, please change.